apps/dashboard/app/models/batch_connect/session.rb

```diff
@@ -307,7 +307,11 @@ def suspended?
     # Whether job is running but still hasn't generated the connection file
     # @return [Boolean] whether starting
     def starting?
-      status.running? && !connect_file.file?
+      if native_host?
+        status.queued?
```

Comment:
Why should this be available when job is queued?

Reply:
Yea I think we can be smarter about what's considered running here. Also I've created https://github.com/OSC/ood_core/issues/229 which would fix what queued state in k8s means.